Runtime/Triangulator.cs

```diff
@@ -661,25 +668,28 @@ public void Execute()
 
                 var min = (float2)float.MaxValue;
                 var max = (float2)float.MinValue;
-
                 for (int i = 0; i < positions.Length; i++)
                 {
+                    if (ignorePositions.IsCreated && ignorePositions[i])
+                        continue;
+                    
                     var p = positions[i];
                     min = math.min(min, p);
                     max = math.max(max, p);
-                    ids[i] = i;
+                    ids.Add(i);
```

Comment:
As context. This loop was probably fully vectorized before the PR. But with the PR, you have introduced three additional branches (two for the IF, and one for the Add method to check if the list needs resizing). This likely makes this particular loop a lot slower. However, it is likely that this loop is not a significant fraction of the run time anyway.
In any case, I'd recommend profiling with/without the PR changes to see if there's any perf impact in the common case.